docs/source/guides/speed.rst

```diff
@@ -160,25 +147,36 @@ For debugging purposes or for dataloaders that load very small datasets, it is d
 
 Spawn
 """""
-When using ``strategy=ddp_spawn`` or training on TPUs, the way multiple GPUs/TPU cores are used is by calling ``.spawn()`` under the hood.
-The problem is that PyTorch has issues with ``num_workers > 0`` when using ``.spawn()``. For this reason we recommend you
-use ``strategy=ddp`` so you can increase the ``num_workers``, however your script has to be callable like so:
+
+When using ``strategy="ddp_spawn"`` or training on TPUs, the way multiple GPUs/TPU cores are used is by calling ``.spawn()`` under the hood.
+The problem is that PyTorch has issues with ``num_workers>0`` when using ``.spawn()``. For this reason we recommend you
+use ``strategy="ddp"`` so you can increase the ``num_workers``, however since DDP doesn't work in interactive environment like ipython/jupyter notebooks
+your script has to be callable like so:
```

Comment:
However, using ddp_spawn enables to reduce memory usage with In-Memory Dataset and shared memory tensors. Worth linking to the PyTorch Geometric example.

Reply:
> Worth linking to the PyTorch Geometric example.

which example? can you share more info?

Reply:
This one: https://pytorch-lightning.readthedocs.io/en/latest/advanced/training_tricks.html#sharing-datasets-across-process-boundaries

Reply:
done!